pages/posts/nepal-2022.md

```diff
@@ -0,0 +1,82 @@
+
+Putting that single bad experience with the locals aside, we went to this place for dinner by the Phewa Lake--everything happens around Phewa Lake in Pokhara--that had good food, a good environment and nive local live music. That was the end of our time in Pokhara, a city I would love to come back to to spend some quality time but by road and not by flight...or so I thought.
+
+The next day as we checked out of the hostel, the adventure sports shop guy called us and asked us whether we want to partake in paragliding. So next thing we know, we were in a cab hailing us from Baidam Road to the top of a hill.
+
+### pokhara -> kathmandu
+The paragliding was a good experience overall but it also meant we were late for the last flight out of Pokhara for the day. We did squander around the airport trying to see if another flight would be scheduled but nope. The next best option was to take a local cab from right outisde the airport to Kathmandu. 
+
+<img src="./../../static/img/posts/nepal/cab-pokhara-accident.png">
+<img src="./../../static/img/posts/nepal/cab-pokhara-inside.png">
+
+So we hopped onto the most normal looking cab and took our seats at the back. It was a 6-seater cab, but the resourceful cabbies loaded upto 10-12 people inside the cab. It was packed to the brim with adults, kids, and lots of luggage. We weren't feeling so good and as luck would have it, a local doctor was also sitting next to us. He slipped us some tablets that he said were for headache. And next thing I personally remember is getting off in Kathmandu at midnight. I was dazed (and later confused) through the whole journey. It was only later that I came to know that the journey was arduous thanks to the terrible roads and that we even had an accident on the way which caused such a delay. Bottomline, there's no good way to travel between Kathmandu and Pokhara, unless of course you're driving on your own.
+
+### kathmandu
+
+Fun fact: my friend carried a frog in one of his spare shoes in his bag from Pokhara(probably Phewa Lake) to Kathmandu. Poor lad must've been overwhelmed from the chaos of the city.
+
+Back in Kathmandu, we stayed in the old town area, and witnessed remnants of the destruction from the [2015 Nepal earthquake](https://en.wikipedia.org/wiki/April_2015_Nepal_earthquake) that rocked the country. The Patan Durbar Square was being renovated along with other structures in and around Durbar Square. But the residential parts of the old town area were exactly that, an old town. You could enter an alley, discover a small temple, while old folks are sitting next to the temple, having a chat or a smoke.
```

Comment:
Yeah but everything was freaking clean, "old town" generally doesn't reflect that, old delhi for example